cql3/query_processor.cc

```diff
@@ -348,6 +348,11 @@ query_processor::query_processor(service::storage_proxy& proxy, database& db, se
                             [] { return prepared_statements_cache::shard_stats().prepared_cache_evictions; },
                             sm::description("Counts the number of prepared statements cache entries evictions.")),
 
+                    sm::make_derive(
+                            "new_gen_on_cache_size_evictions",
+                            [] { return prepared_statements_cache::shard_stats().new_gen_on_cache_size_evictions; },
+                            sm::description("Counts the number of prepared statements cache entries evictions from the new generation partition due to a cache size restrictions. This is usually a sign of a cache pollution.")),
```

Comment:
As I noted before, I think this user-facing counter needs an easier to understand explanation, not using the phrases "new generation" or "usually a sign of cache polution". Why not say something explicit like "Counts the number of prepared statements that were evicted from the prepared-statement cache before being used even once"? (or twice or whatever the threshold really means, I had a question about this above).

Reply:
https://github.com/scylladb/scylla/pull/9591#discussion_r755332307

Reply:
I don't understand, you are saying that you changed the text of this message?
No, renaming "new generation" to "unprivileged" isn't what I'm asking for. A user also won't understand the word "unprivileged". Please read my comment again and check if it really doesn't apply to your newest version (if I doesn't, I don't understand what I'm missing).

Reply:
Sorry. NP. I'll change the description.

Reply:
Pushed a patch that updates the metrics description. Please, review.